src/org/labkey/test/tests/AdminConsoleTest.java

```diff
@@ -68,7 +68,7 @@ public void testRibbonBar()
         waitForElement(Locator.xpath("//div[contains(text(), 'Cannot enable the ribbon message without providing a message to show')]"));
 
         String linkText = "and also click this...";
-        String html = "READ ME!!!  <a href='<%=contextPath%>/project/home/begin.view'>" + linkText + "</a>";
+        String html = "READ ME!!!  <a href='<%=contextPath%>" + "/home/project-begin.view'>" + linkText + "</a>";
```

Comment:
Looks like some structural search weirdness
```suggestion
        String html = "READ ME!!!  <a href='<%=contextPath%>/home/project-begin.view'>" + linkText + "</a>";
```

Reply:
Looks a little funny in this case, but it is intentional. I use `<%=contextPath%>/` to search for controller-first URLs. Path-first URLs usually have `<%=contextPath%><%=containerPath%>`, but obviously not the case of a hard-coded container path like this. Separating like this means the URL won't show up in searches.